clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -62,6 +63,41 @@ namespace drmemtrace {
         }                                  \
     } while (0)
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct pt_packet_decoder_t {
```

Comment:
Maybe put this in a separate header file with a comment that this is derived from the libipt internal struct.

Reply:
Add a link to the libipt code too.